test/unit/auth/auth-api-request.spec.ts

```diff
@@ -3503,12 +3503,20 @@ AUTH_REQUEST_HANDLER_TESTS.forEach((handler) => {
         enabled: true,
         clientId: 'CLIENT_ID',
         issuer: 'https://oidc.com/issuer',
+        clientSecret: 'CLIENT_SECRET',
```

Comment:
I am concerned about legacy customers and would recommend keeping a test without `clientSecret` and `responseType`. Can you add new tests instead of modifying the existing ones?

Reply:
I add tests for createOAuthIdpConfig and updateOAuthIdpConfig